lib/kitchen/driver/opennebula.rb

```diff
@@ -175,13 +175,16 @@ def destroy(state)
 
       def opennebula_connect()        
         opennebula_creds = nil
-        if File.exists?(config[:oneauth_file])
```

Comment:
I don't dislike the idea but I cannot find ONEAUTH_USERNAME and ONEAUTH_PASSWORD in OpenNebula documentation ? I'd rather stay with the env variables defined at https://docs.opennebula.org/5.4/operation/users_groups_management/manage_users.html#shell-environment

Reply:
Fixed